Tests/SourceKitDTests/CrashRecoveryTests.swift

```diff
@@ -48,39 +48,22 @@ final class CrashRecoveryTests: XCTestCase {
     try XCTSkipUnless(Platform.current == .darwin, "Linux and Windows use in-process sourcekitd")
     try XCTSkipIf(longTestsDisabled)
 
-    let ws = try await staticSourceKitTibsWorkspace(name: "sourcekitdCrashRecovery")!
-    let loc = ws.testLoc("loc")
-
-    // Open the document. Wait for the semantic diagnostics to know it has been fully opened and we are not entering any data races about outstanding diagnostics when we crash sourcekitd.
+    let testClient = try await TestSourceKitLSPClient()
+    let uri = DocumentURI.for(.swift)
 
-    try ws.openDocument(loc.url, language: .swift)
+    let positions = testClient.openDocument("""
+      func 1️⃣foo() {
+        print("Hello world")
+      }
+      """, uri: uri)
 
     // Wait for diagnostics to be produced to make sure the document open got handled by sourcekitd.
-    _ = try await ws.testClient.nextDiagnosticsNotification()
-
-    // Make a change to the file that's not saved to disk. This way we can check that we re-open the correct in-memory state.
-
-    let addFuncChange = TextDocumentContentChangeEvent(
-      range: loc.position..<loc.position,
-      rangeLength: 0,
-      text: """
-
-        func foo() {
-          print("Hello world")
-        }
-        """
-    )
-    ws.testClient.send(
-      DidChangeTextDocumentNotification(
-        textDocument: VersionedTextDocumentIdentifier(loc.docUri, version: 2),
-        contentChanges: [addFuncChange]
-      )
-    )
```

Comment:
Do we not need to test for this anymore?

Reply:
Since the entire file is opened in-memory and there’s no on-disk file (because we just use a `TestSourceKitLSPClient` and not e.g. an `IndexedSingleFileSwiftWorkspace`), there’s no on-disk version of the file that sourcekit-lsp could fall back to. So no, need for this anymore.